src/Behavior/DefaultMove.h

```diff
@@ -28,37 +28,37 @@ class DefaultMove : public Movable
 	virtual void AddOnMoveSubscriber(MoveSubscriber& subscriber) override;
 	virtual void RemoveOnMoveSubscriber(MoveSubscriber& subscriber) override;
 	virtual void NotifyOnMoveSubscribers(EventMoveInfo event) override;
-	virtual Vector2 GetPosition() override;
+	virtual glm::vec2 GetPosition() override;
 	virtual float GetX() override;
 	virtual float GetY() override;
-	virtual void SetPosition(Vector2 position, bool emit) override;
+	virtual void SetPosition(const glm::vec2 position, bool emit) override;
 	virtual void SetPosition(float x, float y, bool emit) override;
 	virtual void SetX(float x, bool emit) override;
 	virtual void SetY(float y, bool emit) override;
 	virtual float GetAbsoluteX() override;
 	virtual float GetAbsoluteY() override;
-	virtual Vector2 GetAbsolutePosition() override;
+	virtual glm::vec2 GetAbsolutePosition() override;
 
-	virtual void SetTranslate(Vector2 offset, bool emit) override;
+	virtual void SetTranslate(const glm::vec2 &offset, bool emit) override;
 	virtual void SetTranslateX(float x, bool emit) override;
 	virtual void SetTranslateY(float Y, bool emit) override;
 
-	virtual Vector2 GetTranslate() override;
+	virtual glm::vec2 GetTranslate() override;
 	virtual float GetTranslateX() override;
 	virtual float GetTranslateY() override;
 
-	Vector2 GetChildrenTranslate() const;
-	void TranslateChildren(Vector2 translate);
+	glm::vec2 GetChildrenTranslate() const;
+	void TranslateChildren(const glm::vec2 &translate);
```

Comment:
Original parameter was without a reference. Please remove the reference and pass by copy.